app/javascript/packs/followButtons.js

```diff
@@ -51,23 +51,30 @@ function addButtonFollowText(button, style) {
  */
 function addAriaLabelToButton({ button, followType, followName, style = '' }) {
   let label = '';
+  let pressed = '';
   switch (style) {
     case 'follow':
       label = `Follow ${followType.toLowerCase()}: ${followName}`;
+      pressed = 'false';
       break;
     case 'follow-back':
       label = `Follow ${followType.toLowerCase()} back: ${followName}`;
+      pressed = 'true';
```

Comment:
I think in this case, pressed should be 'false', since the user isn't following them yet